http/src/main/java/com/github/awsjavakit/http/token/OAuthTokenResponse.java

```diff
@@ -0,0 +1,52 @@
+package com.github.awsjavakit.http.token;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+
+public class OAuthTokenResponse {
```

Comment:
You want this as a mutable class, isn't this the prime example of immutability?

Reply:
I will refactor this class in a subsequent PR